pkg/htmlreport/main.go

```diff
@@ -46,9 +46,9 @@ type SigFailure struct {
 }
 
 var jobRegexAliases = map[string]string{
-	"compute": "sig-compute$|sig-compute-serial$|sig-compute-migrations$|sig-operator$|vgpu$|sev$",
-	"network": "sig-network$|sriov$",
-	"storage": "sig-storage$",
+	"compute": "sig-compute|sig-compute-serial|sig-compute-migrations|sig-operator|vgpu|sev",
```

Comment:
this might also pull in other jobs from the main branch. do we want that?

also do we not want to restrict from which release branches we want to pull in the results?

(maybe I am also overthinking here...)

```suggestion
	"compute": "(sig-compute|sig-compute-serial|sig-compute-migrations|sig-operator|vgpu|sev)(-1.[0-9]+)?",
```